src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -31,8 +31,8 @@ azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==1.5.0
 azure-mgmt-containerregistry==3.0.0rc15
 azure-mgmt-containerservice==9.4.0
-azure-mgmt-core==1.2.0
-azure-core==1.7.0
+azure-mgmt-core==1.2.1
+azure-core==1.8.2
```

Comment:
`azure-mgmt-core` already specifies the dependency on `azure-core`.

https://github.com/Azure/azure-sdk-for-python/blob/e1001d7f7d9708b33f087c7a5ac652d1adeca89e/sdk/core/azure-mgmt-core/setup.py#L70

```py
    install_requires=[
        "azure-core<2.0.0,>=1.9.0",
    ],
```

Does CLI need to specify it directly?

Reply:
Besides azure-mgmt-core, data plane commands use azure.core directly, so we need specify the version for it.

Reply:
Installing `azure-mgmt-core` will also install `azure-core`, and `1.8.2` doesn't fit the range `"azure-core<2.0.0,>=1.9.0"`.

Reply:
That makes sense. 

`azure-mgmt-core` 1.2.1 has dependency

https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-core_1.2.1/sdk/core/azure-mgmt-core/setup.py#L70
```py
"azure-core<2.0.0,>=1.8.2",
```

Make sure this dependency is always satisfied if we bump any libs.